components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.scss

```diff
@@ -70,7 +70,7 @@ chef-loading-spinner {
     chef-table-cell:last-child {
       chef-button {
         margin: -0.5em 0;
-        float: right;
+        margin-left: auto;
```

Comment:
this was needed for the get buttons because floats dont work in flex.

Reply:
[this is a total nitpick]: while I don't think there's anything wrong with this, since everything else is going flex, It might be a bit more "flex-like" to use `align-self: flex-end;` here.